package-lock.json

```diff
@@ -5,12 +5,9 @@
     "requires": true,
     "packages": {
         "": {
-            "name": "cubecubed",
             "version": "1.0.8",
             "license": "MIT",
             "dependencies": {
-                "@types/d3": "^7.1.0",
-                "@types/katex": "^0.11.1",
                 "d3": "^7.1.1",
                 "inquirer": "^8.2.0",
                 "katex": "^0.15.1"
```

Comment:
I have some problems with `katex` npm package (text rendering for example). So I'm considering to remove it. What's your idea about this?